modules/pasta.nix

```diff
@@ -0,0 +1,75 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+{
+  options.pasta = {
+    enable = mkEnableOption "Pasta networking" // { default = true; };
```

Comment:
I don't think it should be enabled by default. It changes the network behavior quite a lot.

Reply:
I agree with you that it changes the behavior a lot. I have no problem with disabling it by default.
My main reason for suggesting to enable it, was to have a *secure-by-default* configuration.